packages/manager/apps/dedicated/client/app/components/dedicated-cloud/datacenters/upgrade-range/controller.js

```diff
@@ -10,11 +10,26 @@ export default class {
       init: true,
     };
 
-    return this.DedicatedCloud.getManagementFee(
-      this.productId,
+    return this.DedicatedCloud.getManagementFeePlan(
+      this.managementFeeServiceId,
       this.upgradeCode,
       1,
     )
+      .then((data) => {
+        const { duration, pricingMode } = data.prices?.find(({ capacities }) =>
+          capacities.includes('renew'),
+        );
```

Comment:
I guess this will trigger an `TypeError` exception if:
- prices is undefined/null/empty array

Reply:
Done